Sources/MongoSwift/ClientSession.swift

```diff
@@ -144,21 +144,18 @@ public final class ClientSession {
     }
 
     /// The transaction state of this session.
-    internal var transactionState: TransactionState? {
+    internal var transactionState: TransactionState {
```

Comment:
I don't think we really need the distinction we had here; if we have not started any libmongoc session or have ended the current one then the transaction state for our Swift session should be none.